cli/config/cache.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use function Niden\Core\envValue;
+
+return [
+    'servers'  => [
+        0 => [
+            'host'   => envValue('DATA_API_MEMCACHED_HOST', '127.0.0.1'),
```

Comment:
Can we use the same approach as in Vokuro (with getenv)? I think it will be more clear for our users.

Reply:
`envValue()` uses `getenv`

Reply:
I know, but prefer not to use the wrapper and use the same approach everywhere. 
We could also leave it as it is and refactor this when this is in. https://github.com/phalcon/cphalcon/pull/14339